apps/user/views.py

```diff
@@ -129,6 +130,24 @@ def get_notifications(self, request, pk=None, version=None):
         serializer = self.get_serializer(self.page, many=True)
         return self.get_paginated_response(serializer.data)
 
+    @action(
+        detail=False,
+        permission_classes=[permissions.IsAuthenticated],
+        url_path='me/change-password',
+        serializer_class=PasswordChangeSerializer,
+        methods=['POST']
+    )
+    def change_password(self, request, pk=None, version=None):
+        serializer = PasswordChangeSerializer(
+            data=request.data,
+            context={
+                'request': request
+            }
+        )
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return response.Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
Let's remove `serializer.data` from here. Don't think we need to send the password back.